projects/showcase/cypress/support/e2e-driver.ts

```diff
@@ -1,198 +1,112 @@
-/// <reference types="cypress" />
-import Sinon from 'cypress/types/sinon';
-import pixelmatch from 'pixelmatch';
-import { PNG } from 'pngjs';
+import { PNG } from "pngjs";
+import { MapLibreAssertable, MaplibreCypressHelper } from "./maplibre-helper";
 
 export class E2eDriver {
-  private width: number;
-  private height: number;
-  private referenceImageBuffer: Buffer;
-  private alertStub: Cypress.Agent<Sinon.SinonStub<any[], any>>;
+  private helper = new MaplibreCypressHelper();
 
-  visitMapPage = (url: string): E2eDriver => {
-    cy.visit(url);
-    this.alertStub = cy.stub().as('alertStub');
-    return this;
+  beforeAndAfter = () => {
+    beforeEach(() => {});
```

Comment:
What's the point in those if they are empty?

Reply:
placeholders for others who may wish to add something in the future.  I think we should move the code from cypress commands into here.